### ISSUE TITLE
k8s Events not being created during processing of Triggers

### ISSUE THREAD

**Issue by khrm** [tektoncd/triggers#1602]
# Expected Behavior
k8s should be visible in Triggers Namespace.
# Actual Behavior
k8s events aren't being created.
From the logs we can see:
```
E0605 18:26:21.961947       1 event.go:335] Could not construct reference to: '&v1beta1.EventListener{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"listener", GenerateName:"", Namespace:"default", SelfLink:"", UID:"7b513239-9602-4f1e-96f6-7db922910fec", ResourceVersion:"71208", Generation:1, CreationTimestamp:time.Date(2023, time.June, 5, 14, 50, 8, 0, time.Local), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry{v1.ManagedFieldsEntry{Manager:"kubectl-create", Operation:"Update", APIVersion:"triggers.tekton.dev/v1beta1", Time:time.Date(2023, time.June, 5, 14, 50, 8, 0, time.Local), FieldsType:"FieldsV1", FieldsV1:(*v1.FieldsV1)(0xc0004fa528), Subresource:""}, v1.ManagedFieldsEntry{Manager:"controller", Operation:"Update", APIVersion:"triggers.tekton.dev/v1beta1", Time:time.Date(2023, time.June, 5, 18, 26, 8, 0, time.Local), FieldsType:"FieldsV1", FieldsV1:(*v1.FieldsV1)(0xc0004fa558), Subresource:"status"}}}, Spec:v1beta1.EventListenerSpec{ServiceAccountName:"tekton-triggers-example-sa", Triggers:[]v1beta1.EventListenerTrigger{v1beta1.EventListenerTrigger{Bindings:[]*v1beta1.TriggerSpecBinding{(*v1beta1.TriggerSpecBinding)(0xc0003eafa0), (*v1beta1.TriggerSpecBinding)(0xc0003eaff0)}, Template:(*v1beta1.TriggerSpecTemplate)(0xc0006c5040), TriggerRef:"", Name:"foo-trig", Interceptors:[]*v1beta1.TriggerInterceptor(nil), ServiceAccountName:""}}, TriggerGroups:[]v1beta1.EventListenerTriggerGroup(nil), NamespaceSelector:v1beta1.NamespaceSelector{MatchNames:[]string(nil)}, LabelSelector:(*v1.LabelSelector)(nil), Resources:v1beta1.Resources{KubernetesResource:(*v1beta1.KubernetesResource)(nil), CustomResource:(*v1beta1.CustomResource)(nil)}, CloudEventURI:""}, Status:v1beta1.EventListenerStatus{Status:v1.Status{ObservedGeneration:0, Conditions:v1.Conditions{apis.Condition{Type:"Available", Status:"True", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2023, time.June, 5, 17, 45, 11, 0, time.Local)}, Reason:"MinimumReplicasAvailable", Message:"Deployment has minimum availability."}, apis.Condition{Type:"Deployment", Status:"True", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2023, time.June, 5, 18, 26, 8, 0, time.Local)}, Reason:"", Message:"Deployment exists"}, apis.Condition{Type:"Progressing", Status:"True", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2023, time.June, 5, 18, 26, 8, 0, time.Local)}, Reason:"ReplicaSetUpdated", Message:"ReplicaSet \"el-listener-74df8bf4bf\" is progressing."}, apis.Condition{Type:"Ready", Status:"True", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2023, time.June, 5, 18, 26, 8, 0, time.Local)}, Reason:"", Message:"EventListener is ready"}, apis.Condition{Type:"Service", Status:"True", Severity:"", LastTransitionTime:apis.VolatileTime{Inner:time.Date(2023, time.June, 5, 18, 26, 8, 0, time.Local)}, Reason:"", Message:"Service exists"}}, Annotations:map[string]string(nil)}, AddressStatus:v1beta1.AddressStatus{Address:(*v1beta1.Addressable)(0xc000012078)}, Configuration:v1beta1.EventListenerConfig{GeneratedResourceName:"el-listener"}}}' due to: 'no kind is registered for the type v1beta1.EventListener in scheme "pkg/runtime/scheme.go:100"'. Will not report event: 'Normal' 'dev.tekton.event.triggers.done.v1' ''

```

# Steps to Reproduce the Problem

1. put `enable` instead of `disable` in `el-events` of controller.yaml
2. Send an event/trigger/webhook to EL.
3. You will see the logs given above and no k8s Events.

# Additional Info

Triggers version: v0.24.0

**Comment by khrm** [comment]
Fixed by #1603

**Comment by dibyom** [comment]
@khrm can we close this now?
